src/Container/AbstractObjectContainer.php

```diff
@@ -27,43 +27,43 @@ public function offsetExists($offset): bool
     }
 
     #[\ReturnTypeWillChange]
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
     {
         $element = array_slice($this->objects, $offset, 1);
 
         return $this->offsetExists($offset) ? array_shift($element) : null;
     }
 
     #[\ReturnTypeWillChange]
-    public function offsetSet($offset, $value)
+    public function offsetSet($offset, $value): mixed
     {
         throw new InvalidArgumentException('This method can\'t be used as object are stored with a string as array index', __LINE__);
     }
 
     #[\ReturnTypeWillChange]
-    public function offsetUnset($offset)
+    public function offsetUnset($offset): void
     {
         if ($this->offsetExists($offset)) {
             unset($this->objects[$this->getObjectKey($this->offsetGet($offset))]);
         }
     }
 
     #[\ReturnTypeWillChange]
-    public function current()
+    public function current(): mixed
```

Comment:
`mixed` is available only since PHP >= 8.0, the project is usable from PHP >= 7.4